curriculum/challenges/english/01-responsive-web-design/css-nutrition-label/step-048.md

```diff
@@ -0,0 +1,153 @@
+---
+id: 615f7ad94380408d971d14f6
+title: Step 48
+challengeType: 0
+dashedName: step-48
+---
+
+# --description--
+
+The `:not` pseudo-selector can be used to select all elements that do not match the given CSS rule. 
+
+Modify your `.daily-value p` selector to exclude the `.no-divider` elements.
```

Comment:
Not entirely necessary, because the hint is perfect. However, I am not sure Campers have encountered this yet.

````suggestion
The `:not` pseudo-selector can be used to select all elements that do not match the given CSS rule. 

```css
div:not(#example) {
  color: red;
}
```

The above selects all `div` elements without an `id` of `example`.

Modify your `.daily-value p` selector to exclude the `.no-divider` elements.
````